base/docs/basedocs.jl

```diff
@@ -447,6 +447,22 @@ julia> A'
 """
 kw"'"
 
+"""
+    \$
+
+The string interpolation operator, see [string](@ref string-interpolation and [expression](@ref man-expression-interpolation).
```

Comment:
I think we would need more details here :sweat_smile: 

You can have a look at docstrings of `=` or other operators.